### DISCUSSION TITLE
Low Estimated Payout

### DISCUSSION THREAD

**Discussion by Eomm** [community/community#66282] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,
I got a monthly subscriber (5$/month), and now I can see this value in my GH Sponsorship menu:

<img width="348" alt="image" src="https://github.com/community/community/assets/11404065/ec4e9bee-4b5a-4599-add0-7dd97c15af0c">

The sponsorship:

<img width="491" alt="image" src="https://github.com/community/community/assets/11404065/28fafbab-ad53-48c1-9d21-826ab0b92263">

Stripe view:

<img width="639" alt="image" src="https://github.com/community/community/assets/11404065/742734cc-f9f7-485a-887a-95d5b7ee0828">

Anybody can help me understand what is going on?
Why the payout is so small?

I faced the same situation in February, but the payout was "quite right":

![image](https://github.com/community/community/assets/11404065/6328fa0d-db6b-44ff-8bac-67cd6fd0e8e8)

I received already some bucks, but it is the first time it is so non-sense..

**Comment by vbarbar** [comment, marked as answer]
As mentioned in the question, the payout amount in your local currency may be lower than expected due to the currency exchange rate used by GitHub.

GitHub uses the exchange rate at the time of payout to convert the USD sponsorship amount to your local currency. If the exchange rate has changed since you viewed the estimated payout amount, it could result in a lower payout.
